src/media/css/app-list.styl

```diff
@@ -106,6 +103,31 @@ $desktop-tile-padding = 20px 50px;
     }
 }
 
+.filters-content {
+    height: 40px;
+    line-height: 40px;
+}
+
+.expand-toggle {
```

Comment:
serious points if you rename this to .app-list-filters-expand-toggle :)

it's long...but it'll follow closely our future class naming scheme and it's nicely explicit